KomodoPlatform/atomicDEX-Desktop
# Implement debug CI builds

To test:
- download artefacts from https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/3311154117
- Confirm app will launch, login and do rick/morty swap on all OS

Please post which OS you have confirmed in this thread.

@SylEze if it looks ok, you can merge into `fix/loop_crashes` for more testing on debug.
